src/routes/AuthLayoutRouter/AuthLayoutRouter.js

```diff
@@ -370,7 +376,7 @@ export default function AuthLayoutRouter (props) {
           <div styleName={cx('detail', { hidden: !hasDetail })} id={DETAIL_COLUMN_ID}>
             {/* NOTE: These routes could potentially be simply: `(.*)/${POST_DETAIL_MATCH}` and`(.*)/${GROUP_DETAIL_MATCH}` */}
             <Switch>
-              <Route path={`/:context(all|public)/:view(events|explore|groups|map|projects|stream)/${POST_DETAIL_MATCH}`} component={PostDetail} />
+              <Route path={`/:context(all|public|my)/:view(events|explore|groups|map|projects|stream|mentions|interactions|posts|announcements)/${POST_DETAIL_MATCH}`} component={PostDetail} />
```

Comment:
i would split this up, because this would match /all/mentions which isnt a thing
so the /my stuff should go on its own line i think

Reply:
yep, ok